feature/bgp/addpath/ate_tests/route_propagation_test/route_propagation_test.go

```diff
@@ -154,9 +154,43 @@ func TestBGP(t *testing.T) {
 		desc:     "propagate IPv4 over IPv4",
 		fullDesc: "Advertise prefixes from ATE port1, observe received prefixes at ATE port2",
 		dut: dutData{&telemetry.NetworkInstance_Protocol_Bgp{
+			PeerGroup: map[string]*telemetry.NetworkInstance_Protocol_Bgp_PeerGroup{
+				"BGP-PEER-GROUP1": {
+					PeerGroupName: ygot.String("BGP-PEER-GROUP1"),
+					AfiSafi: map[telemetry.E_BgpTypes_AFI_SAFI_TYPE]*telemetry.NetworkInstance_Protocol_Bgp_PeerGroup_AfiSafi{
+						telemetry.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST: {
```

Comment:
Do we still need the group level afi-safi configured given that the same is configured on the global level? Consider removing group level ones... Same for the remaining.

Reply:
@Hobbydos Please confirm.

Reply:
@jjing1122 cc @vcheny 
Peer group level afi-safi configs were removed in favor for global level ones;
commit removed c0cb515
in latest commit, you'll only see global and targeted neighbor configs such as 
https://github.com/openconfig/featureprofiles/blob/a39ddda34a80c6f697fb91e918f32a3c452ef78f/feature/bgp/addpath/ate_tests/route_propagation_test/route_propagation_test.go#L198-L207